src/integrationTest/java/org/opensearch/security/ResourceFocusedTests.java

```diff
@@ -231,37 +188,4 @@ private byte[] createCompressedRequestBody(final RequestBodySize size) {
             throw new RuntimeException(ioe);
         }
     }
-
```

Comment:
qq: Curious to know why stats printer is removed?

Reply:
It was disabled in code, effectively commented out.  It was also a human person to run the test case and watch the log - which when authoring the test was useful, at this point not so much.